caffe2/python/data_parallel_model.py

```diff
@@ -205,6 +211,15 @@ def Parallelize_GPU(
         devices, model_helper_obj, model_helper_obj.param_init_net, sync_names
     )
 
+    # Handle any operations that need to be done after parameter sync
+    # i.e. making sure multi-precision copies of parameters are up-to-date
+    if post_sync_builder_fun is not None:
```

Comment:
@slayton58  looks like you don't use the post_sync_builder_fun anymore? Perhaps because optimizer_builder_fun is now possible? 

In that case, let's remove it.